src/index.ts

```diff
@@ -68,6 +73,13 @@ export async function certificateFor(domain: string, options: Options = {}) {
   }
 
   debug(`Returning domain certificate`);
+  if (options.returnCa) {
+    return {
+      ca: options.returnCa === 'read' ? readFile(rootCACertPath) : rootCACertPath,
```

Comment:
Per my request for an API change, how about we change this to add a `caPath` property:

```suggestion
      ca: options.getCaBuffer && readFile(rootCACertPath),
      caPath: options.getCaPath && rootCACertPath,
```

Reply:
I like it.  I will need to update the condition, and the jsdoc before moving forward, though